### DISCUSSION TITLE
Serial Install

### DISCUSSION THREAD

**Discussion by ddominet** [Security-Onion-Solutions/securityonion#6638] [Unsupported Versions] [answered]
Hi, is it possible to install via serial console ? 
I was planning on trying it out on an appliance without VGA card

**Comment by dougburks** [comment, marked as answer]
I don't know that we've ever tried installing via serial console. Our installer is text based, so please feel free to try it and let us know how it goes.

**Reply by ukr42** [reply]
I tried the other day to install via USB stick on a system that has only a serial console port. I can't get past the initial line "isolinux ...", then it's stuck.

I did a successful installation of the opnsense serial image, but all other distros I tried (vanilla centos, ubuntu,...) show the same behaviour. The problem seems to be that the grub menu is send to the standard console and is therefore inaccessible - otherwise editing the boot-entry would probably be the way to go.

Another way could be a remastered image, either with a modified grub configuration or an automatic installation with preseed.cfg. Only tried the first one and failed miserably ;)

**Comment by ukr42** [comment]
Works with unetbootin (Ubuntu 18 or 20):
https://github.com/ynkjm/ubuntu-serial-install/blob/master/README.md
